system/Model.php

```diff
@@ -737,6 +750,13 @@ public function __call(string $name, array $params)
         return $result;
     }
 
+    private function checkBuilderMethod(string $name): void
```

Comment:
Not 100% sure of the current docblock guidelines, other than I know they were updated recently, but doesn't this need a docblock?

Reply:
Added docblock.